lib/ConditionUtils/core/Utils.py

```diff
@@ -10,14 +10,17 @@
 
 from DataFileUtil.DataFileUtilClient import DataFileUtil
 from KBaseSearchEngine.KBaseSearchEngineClient import KBaseSearchEngine
+from GenericsAPI.GenericsAPIClient import GenericsAPI
 
 
 class Utils:
     def __init__(self, config):
         self.cfg = config
         self.scratch = config['scratch']
-        self.dfu = DataFileUtil(os.environ['SDK_CALLBACK_URL'])
+        self.callback_url = os.environ['SDK_CALLBACK_URL']
+        self.dfu = DataFileUtil(self.callback_url)
         self.kbse = KBaseSearchEngine(config['search-url'])
+        self.gen_api = GenericsAPI(self.callback_url, service_ver='dev')
```

Comment:
I'm not sure we want to pin to the dev version. I'd rather not specify (should get the dev if nothing else is yet available)

Reply:
👍